src/api/resources.ts

```diff
@@ -35,3 +35,12 @@ export async function dislike(id: number) {
 
   return result
 }
+
+export async function get(id: number) {
```

Comment:
There's a type mismatch between this expecting a number and the loader casting the resourceId as string, which is causing the preview build to fail when it transpiles the Typescript.